sql/parser/sql.y

```diff
@@ -54,8 +55,8 @@ package parser
   limit          *Limit
   targetList     TargetList
   targetListPtr  *TargetList
-  privilegeType  PrivilegeType
-  privilegeList  PrivilegeList
+  privilegeType  structured.Privilege
```

Comment:
I'd like the constants to stay in the parser package. I'd rather have structured depend on sql/parser. How about we implement #1917?